src/core/AutomationPattern.cpp

```diff
@@ -486,7 +499,10 @@ void AutomationPattern::flipX( int length )
 			{
 				tempValue = valueAt( ( iterate + i ).key() );
 				MidiTime newTime = MidiTime( length - ( iterate + i ).key() );
-				tempMap[newTime] = tempValue;
+				// TODO: Can we be sure newTime will never repeat its value?
```

Comment:
What does this comment mean?

Reply:
I think this can be ignored. I guess being used to working with pointers I made confusion thinking it would be a problem to have `tempMap[newTime] = AutomationNode( tempValue );` if there was already an automation node on that particular time. That's probably not an issue, because if there's already an automation node there it will just run its copy constructor right? I didn't think the algorithm through enough to find out if `newTime` can repeat itself during that sorting, but even if it does it wouldn't be a problem then, I'll just remove that comment and the other similar one on the next commit!